tests/native-lib/pass/ptr_read_access.rs

```diff
@@ -1,8 +1,5 @@
 //@revisions: trace notrace
-//@[trace] only-target: linux
-//@[trace] only-target: gnu
-//@[trace] only-target: x86
-//@[trace] only-on-host
+//@[trace] only-target: x86_64-unknown-linux-gnu x86-unknown-linux-gnu
```

Comment:
What is `x86-unknown-linux-gnu`? There is no such target. Do you mean `i686-unknown-linux-gnu`?

Reply:
facepalm